src/sql/workbench/parts/notebook/cellViews/linkHandler.directive.ts

```diff
@@ -5,19 +5,27 @@
 
 import { Directive, Inject, HostListener, Input } from '@angular/core';
 
+import * as types from 'vs/base/common/types';
 import { URI } from 'vs/base/common/uri';
 import { IOpenerService } from 'vs/platform/opener/common/opener';
 import { onUnexpectedError } from 'vs/base/common/errors';
 import product from 'vs/platform/product/node/product';
+import { INotebookService } from 'sql/workbench/services/notebook/common/notebookService';
 
 const knownSchemes = new Set(['http', 'https', 'file', 'mailto', 'data', `${product.urlProtocol}`, 'azuredatastudio', 'azuredatastudio-insiders', 'vscode', 'vscode-insiders', 'vscode-resource']);
 @Directive({
 	selector: '[link-handler]',
 })
 export class LinkHandlerDirective {
-
+	private workbenchFilePath: URI;
 	@Input() isTrusted: boolean;
-	constructor(@Inject(IOpenerService) private readonly openerService: IOpenerService) {
+	@Input() notebookUri: URI;
+
+	constructor(
+		@Inject(IOpenerService) private readonly openerService: IOpenerService,
+		@Inject(INotebookService) private readonly notebookService: INotebookService
+	) {
+		this.workbenchFilePath = URI.parse(require.toUrl('vs/code/electron-browser/workbench/workbench.html'));
```

Comment:
It should do - there's logic for this in other parts of the code (I took the exact string), but I do want to double-check it once it's in the build today in case there's some weirdness.